devices/Mcp7940xx/Mcp7940xx.nuspec

```
@@ -0,0 +1,37 @@
+<?xml version="1.0" encoding="utf-8"?>
+<package xmlns="http://schemas.microsoft.com/packaging/2012/06/nuspec.xsd">
+  <metadata>
+    <id>nanoFramework.Iot.Device.Mcp7940xx</id>
+    <version>$version$</version>
+    <title>nanoFramework.Iot.Device.Mcp7940xx</title>
+    <authors>nanoFramework project contributors</authors>
+    <owners>nanoFramework,dotnetfoundation</owners>
+    <requireLicenseAcceptance>false</requireLicenseAcceptance>
+    <license type="file">LICENSE.md</license>
+    <releaseNotes>
+    </releaseNotes>
+    <readme>docs\README.md</readme>
+    <developmentDependency>false</developmentDependency>
+    <projectUrl>https://github.com/nanoframework/nanoFramework.IoT.Device</projectUrl>
+    <icon>images\nf-logo.png</icon>
+    <repository type="git" url="https://github.com/nanoframework/nanoFramework.IoT.Device" commit="$commit$" />
+    <copyright>Copyright (c) .NET Foundation and Contributors</copyright>
+    <description>This package includes the .NET IoT Core binding Iot.Device.Mcp7940xx for .NET nanoFramework C# projects.</description>
+    <summary>Iot.Device.Mcp7940xx assembly for .NET nanoFramework C# projects</summary>
+    <tags>nanoFramework C# csharp netmf netnf Iot.Device.Mcp7940xx</tags>
+    <dependencies>
+      <dependency id="nanoFramework.CoreLibrary" version="1.12.0" />
+      <dependency id="nanoFramework.Device.OneWire" version="1.3.4" />
```

```suggestion
      <dependency id="nanoFramework.Iot.Device.Common.NumberHelper" version="1.1.113.2032" />
      <dependency id="nanoFramework.Runtime.Events" version="1.10.0" />
      <dependency id="nanoFramework.System.Device.Gpio" version="1.0.4" />
      <dependency id="nanoFramework.System.Device.I2c" version="1.0.3" />
      <dependency id="nanoFramework.System.Math" version="1.4.4" />
```

You need to have the right nugets references in the package. It should build better ;-)